packages/gatsby/content/advanced/pnp-data.md

```diff
@@ -205,13 +213,15 @@ Note: The algorithm described here is quite inefficient. You should make sure to
 
 3. Let `relativeUrl` be the relative path between `manifest` and `moduleUrl`
 
-    1. Note: Make sure it always starts with a `./` or `../`
+    1. Note: The relative path must not start with `./`; trim it if needed
 
 4. If `relativeUrl` matches `manifest.ignorePatternData`, then
 
     1. Return **null**
 
-5. For each `referenceMap` value in `manifest.packageRegistryData`
+5. Let `relativeUrlWithDot` be `relativeUrl` prefixed with `./` or `../` as necessary
```

Comment:
FYI `relativeUrlWithDot` is never used. I'm assuming it's supposed to be used instead of `relativeUrl` below.